models/apple_search_ads__ad_group_report.sql

```diff
@@ -18,7 +23,7 @@ with ad_group_report as (
         campaign.campaign_name, 
         ad_group.ad_group_id,
         ad_group.ad_group_name,
-        ad_group_report.date_day,
+        ad_group_report.date_day, --
```

Comment:
Can you have `date_day` actually be the first field selected. It makes initial querying of the data way easier to understand right away and helps with coming to grips with the grain if you didn't read our descriptions haha